package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
```

Comment:
Hi! Basically, we don't need it. Please remove.

Reply:
removed the same!